smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/middlewares/ContentLengthMiddleware.kt

```diff
@@ -18,9 +18,9 @@ class ContentLengthMiddleware : MiddlewareRenderable {
     override val position = MiddlewarePosition.BEFORE
 
     override fun render(
-        ctx: ProtocolGenerator.GenerationContext,
+        model: Model,
+        symbolProvider: SymbolProvider,
```

Comment:
how can you change this when I thought the MiddlewareRenderable still has context as a parameter? did it already change and I missed it?